### PR TITLE
Improve the readme

### PR DESCRIPTION
**Pull request by zeroual**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,80 @@ is an application that aims to help people starving all around the wlorld get fr
 ## Getting started
```
Comment (NaitYoussef) [inline README.md:3]: In the above section (Code solidaire) can you please correct 'wlorld' ;)

**Review by NaitYoussef — COMMENTED** [review]
Very useful information, great job

File: README.md (hunk 1 above)
Comment (zeroual) [reply README.md:3]: done

**Comment by zeroual** [comment]
@NaitYoussef  if ok for you approve this PR and merge it

**Review by NaitYoussef — APPROVED** [review]

**Pull request merged**